### ISSUE TITLE
clearAllBodyScrollLocks stops working on 3.1.3

### ISSUE THREAD

**Issue by elrumordelaluz** [willmcpo/body-scroll-lock#196]
Hi!
For some reason updating from `3.0.3` into `3.1.3` seems that `clearAllBodyScrollLocks` stops working as expected. I am using in an `useEffect` callback, something like:

```js
useEffect(() => {
    if (value) disableBodyScroll(ref.current)
    return () => {
      clearAllBodyScrollLocks()
    }
  }, [value])
```

Rolling back into `3.0.3` everything works well.

**Comment by diachedelic** [comment]
Hi, can you tell us what device/browser you are using when you experience this?

**Comment by elrumordelaluz** [comment]
Same behaviour in both `Chrome Version 85.0.4183.83 (Official Build) (64-bit)` and `Firefox 80.0.1 (64-bit)`.

**Comment by fernandoabolafio** [comment]
Same here: Chrome Version 84.0.4147.135 (Official Build) (64-bit)

**Comment by diachedelic** [comment]
Possibly fixed in #199

**Comment by jaimemnds** [comment]
It looks like both `clearAllBodyScrollLocks` and `enableBodyScroll` stopped working in my case.
I've rolled back to `3.0.3` and also got both of them working again. My usage is pretty much the same as @elrumordelaluz's.

**Comment by MylesKingsnorth** [comment]
I also can confirm `clearAllBodyScrollLocks ` and `enableBodyScroll` doesn't work in a `useEffect` on 3.1.3 but does on 3.0.3

**Comment by diachedelic** [comment]
Could somebody please try applying the patch in #199 and see if that fixes it?

**Comment by MylesKingsnorth** [comment]
> Could somebody please try applying the patch in #199 and see if that fixes it?

Seems to work for me

**Comment by pomacs-store** [comment]
published v3.1.5 with #199 change - hopefully fixes it

**Comment by willmcpo** [comment]
Published 3.1.5 with #199  - hopefully fixes it

**Comment by KKasabov** [comment]
> Published 3.1.5 with #199 - hopefully fixes it
Fixed it for me, thanks!
